src/script/components/Conversation/Conversation.tsx

```diff
@@ -460,6 +464,19 @@ export const Conversation = ({
     [addReadReceiptToBatch, repositories.conversation, repositories.integration, updateConversationLastRead],
   );
 
+  const onGoToLastMessage = () => {
```

Comment:
This is a concrete implementation of a handler. The name should not start with `on` it should have a imperative naming (explaining what it does exactly). 

I'd suggest `loadLastReceivedMessages` or something along those lines

Reply:
done, I called it `jumpToLastMessage`